ngraph_bridge/ngraph_builder.cc

```diff
@@ -2675,6 +2675,7 @@ const static std::map<
         {"TopKV2", TranslateTopKV2Op},
         {"Transpose", TranslateTransposeOp},
         {"Unpack", TranslateUnpackOp},
+        {"Where", TranslateSelectOp},
```

Comment:
Please add a test for where op or enable TF test.

Reply:
I assumed that `tests/python/test_select.py` is actually testing for Where since it uses `tf.where`. Let me check again and add explicit Where tests if that's not the case.

Reply:
Added and enabled tests for where op.